src/include/sof/samples/audio/c_models_inc/preprocess.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Mihai Despotovici <mihai.despotovici@nxp.com>
+ */
+
+#ifndef __PREPROCESS_H__
+#define __PREPROCESS_H__
+
+#include <stdint.h>
+
+int preprocess(
+	const int16_t *input,
+	uint8_t *output
+	);
```

Comment:
Do we need to split into multiple lines here? Remember, maximum width of 100 characters.